arm-storsimple8000series/2017-06-01/swagger/storsimple.json

```diff
@@ -0,0 +1,1781 @@
+          "type": "string"
+        }
+      }
+    },
+    "ManagerIntrinsicSettings": {
+      "description": "Intrinsic settings which refers to the type of the Storsimple manager",
+      "required": [
+        "type"
+      ],
+      "type": "object",
+      "properties": {
+        "type": {
+          "description": "Refers to the type of the StorSimple Manager",
+          "enum": [
+            "GardaV1",
+            "HelsinkiV1"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ManagerType",
+            "modelAsString": false
+          }
+        }
+      }
+    },
+    "ManagerList": {
+      "description": "List of storsimple Managers under a particular resourceGroup",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List of Managers",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Manager"
+          }
+        }
+      }
+    },
+    "ManagerProperties": {
+      "description": "The properties of the Manager",
+      "type": "object",
+      "properties": {
+        "cisIntrinsicSettings": {
+          "$ref": "#/definitions/ManagerIntrinsicSettings",
+          "description": "Specifies if the Manager is Garda or Helsinki"
+        },
+        "sku": {
+          "$ref": "#/definitions/ManagerSku",
+          "description": "Specifies the Sku"
+        },
+        "provisioningState": {
+          "description": "Specifies the state of the resource as it is getting provisioned. Value of \"Succeeded\" means the Manager was successfully created",
+          "type": "string"
+        }
+      }
+    },
+    "ManagerSku": {
+      "description": "The Sku.",
+      "required": [
+        "name"
+      ],
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Refers to the sku name which should be \"Standard\"",
+          "enum": [
+            "Standard"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ManagerSkuType",
+            "modelAsString": false
+          }
+        }
+      }
+    },
+    "NetworkInterfaceData0Settings": {
+      "description": "Network interface Data 0 IP settings",
+      "type": "object",
+      "properties": {
+        "controllerZeroIp": {
+          "description": "Controller 0's IPv4 address",
+          "type": "string"
+        },
+        "controllerOneIp": {
+          "description": "Controller 1's IPv4 address",
+          "type": "string"
+        }
+      }
+    },
+    "PublicKey": {
+      "description": "The public key.",
+      "required": [
+        "key"
+      ],
+      "type": "object",
+      "properties": {
+        "key": {
+          "description": "The key",
+          "type": "string"
+        }
+      }
+    },
+    "Resource": {
```

Comment:
@dsgouda can you please take a look at the ResourceModelValidation failures for this PR (or please route as appropriate, I don't know who owns these linter rules)?  It looks to me that Resource is correctly defined.